src/components/scene/inspector.js

```diff
@@ -0,0 +1,52 @@
+/* global AFRAME */
+var registerComponent = require('../../core/component').registerComponent;
+var pkg = require('../../../package');
+
+var INSPECTOR_URL = pkg.homepage + 'releases/' + pkg.version + '/aframe-inspector.min.js';
+
+module.exports.Component = registerComponent('inspector', {
+  schema: {
+    url: {default: INSPECTOR_URL}
+  },
+
+  init: function () {
+    this.onKeydown = this.onKeydown.bind(this);
+  },
+
+  play: function () {
+    window.addEventListener('keydown', this.onKeydown);
+  },
+
+  pause: function () {
+    window.removeEventListener('keydown', this.onKeydown);
+  },
+
+  onKeydown: function (evt) {
+    // Alt + Ctrl + e
```

Comment:
Yep probably that will match better for `inspector` as similar as devtools, and leave `e` for the incoming VR-editor